src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -59,9 +60,11 @@ abstract class CompilerFleetMessageReceiver : MessageReceiver {
     abstract fun processMessage(message: PubsubMessage)
 }
 
+private val taskReceiverLogger = Logger.getLogger("TaskReceiver")
```

Comment:
Use slf4j here as well? And rename to `LOGGER`

Reply:
Done